crates/nu-command/src/filesystem/watch.rs

```diff
@@ -33,6 +37,16 @@ impl Command for Watch {
         vec!["watcher", "reload", "filesystem"]
     }
 
+    fn deprecation_info(&self) -> Vec<DeprecationEntry> {
+        vec![DeprecationEntry {
+            ty: DeprecationType::Flag("--debounce-ms".into()),
+            report_mode: ReportMode::FirstUse,
+            since: Some("0.106.0".into()),
+            expected_removal: Some("0.108.0".into()),
```

Comment:
```suggestion
            since: Some("0.107.0".into()),
            expected_removal: Some("0.109.0".into()),
```
Since this PR wasn't in the 0.106.0 release.

Reply:
Good catch! I don't have access to my laptop until the weekend. Can anyone push this into my branch?